app/ts/simulation/services/EthereumClientService.ts

```diff
@@ -192,8 +195,65 @@ export class EthereumClientService {
 	}
 
 	public readonly multicall = async (transactions: readonly EthereumUnsignedTransaction[], blockNumber: bigint) => {
+		const httpsRpc = this.requestHandler.getRpcNetwork().httpsRpc
+		if (httpsRpc === 'https://rpc.dark.florist/winedancemuffinborrow' || httpsRpc === 'https://rpc.dark.florist/birdchalkrenewtip') {
+			//TODO: Remove this when we get rid of our old multicall
+			return this.executionSpec383MultiCallOnlyTransactions(transactions, blockNumber)
+		}
+
 		const blockAuthor: bigint = MOCK_ADDRESS
 		const unvalidatedResult = await this.requestHandler.jsonRpcRequest({ method: 'eth_multicall', params: [blockNumber, blockAuthor, transactions] })
 		return MulticallResponse.parse(unvalidatedResult)
 	}
+
+	public readonly executionSpec383MultiCall = async (calls: readonly BlockCalls[], blockTag: EthereumBlockTag) => {
+		const call = { method: 'eth_multicallV1', params: [calls, blockTag] } as const
+		console.log(calls)
+		console.log(JSON.stringify(ExecutionSpec383MultiCallParams.serialize(call)))
+		const unvalidatedResult = await this.requestHandler.jsonRpcRequest(call)
+		console.log([0, calls, blockTag])
+		console.log(unvalidatedResult)
```

Comment:
Recommend adding a TODO to remove these log lines, since they appear to just for debugging.  this way you can just search all files for `TODO` to easily find them, rather than having to dig around trying to find all of the stuff that needs to be removed.

It may be valuable to have a comment prefix specific to the multicall stuff like `// MULTICALL TODO: ...` so you can easily find it all.